### DISCUSSION TITLE
2021-47: SSH 弃用 SHA-1 哈希算法

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#305] [Announcements]
# 2021-47: SSH 弃用 SHA-1 哈希算法

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2021-47/

**Comment by noirgif** [comment]
一点附加信息：

根据[这个 stackoverflow](https://security.stackexchange.com/questions/226131/openssh-declares-ssh-rsa-deprecated-what-do-i-do-next)，现在的 ssh-rsa 默认生成的应该是 SHA-256（从7.2起），所以只要在那之后生成的 ssh-rsa key 可以照常用。
